src/Modules/Render/ViewScene.h

```diff
@@ -158,6 +159,14 @@ namespace Render {
     void syncMeshComponentFlags(const std::string& connectedModuleId, Dataflow::Networks::ModuleStateHandle state);
 
     ActiveGeometryMap activeGeoms_;
+
+    class ScopedExecutionReporter
```

Comment:
```suggestion
    class SCISHARE ScopedExecutionReporter
```

I'm not sure this is necessary for nested local classes but it wouldn't hurt